src/overlays/actors/ovl_Bg_Jya_Bombiwa/z_bg_jya_bombiwa.c

```diff
@@ -65,17 +65,20 @@ static InitChainEntry sInitChain[] = {
 void BgJyaBombiwa_SetupDynaPoly(BgJyaBombiwa* this, PlayState* play, CollisionHeader* collision, s32 flag) {
     s16 pad1;
     CollisionHeader* colHeader = NULL;
-    s16 pad2;
 
     DynaPolyActor_Init(&this->dyna, flag);
     CollisionHeader_GetVirtual(collision, &colHeader);
     this->dyna.bgId = DynaPoly_SetBgActor(play, &play->colCtx.dyna, &this->dyna.actor, colHeader);
+
+#if OOT_DEBUG
     if (this->dyna.bgId == BG_ACTOR_MAX) {
+        s16 pad2;
```

Comment:
```suggestion
        s32 pad2;
```
I assume this matches?
(for consistency with the others to help making a hypothetical macro in the future)

Reply:
Yes, and I missed an s16 pad earlier in this function too